src/renderer/components/+apps-releases/release-details.tsx

```diff
@@ -37,6 +38,8 @@ interface Props {
 export class ReleaseDetails extends Component<Props> {
   @observable details: IReleaseDetails;
   @observable values = "";
+  @observable valuesLoading = false;
+  @observable userSuppliedOnly = true;
```

Comment:
To keep current default behaviour this should be false initially

Reply:
Sure, I think it's stale by now, I'll update it soon when I have time, maybe this weekend. And after that could you review it for me soon so this could become upstream.

Reply:
I can do these changes if you would like and don't mind me force pushing to your branch

Reply:
Ok, I appreciate your help, I'm busy right now so I'm couldn't